### PR TITLE
[11.0][IMP] product_secondary_unit: Add security group

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: product_secondary_unit/migrations/11.0.1.0.2/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#  Copyright 2019 Tecnativa - Sergio Teruel
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    group_secondary_unit = env.ref(
+        'product_secondary_unit.group_secondary_unit')
+    users = env['res.users'].search([])
```
Comment (pedrobaeza) [inline product_secondary_unit/migrations/11.0.1.0.2/post-migration.py]: Add `with_context(active_test=False)`

**Review by pedrobaeza — COMMENTED** [review]
Maybe you should add the group in settings screen

File: product_secondary_unit/migrations/11.0.1.0.2/post-migration.py (hunk 1 above)
Comment (sergio-teruel) [reply product_secondary_unit/migrations/11.0.1.0.2/post-migration.py]: Why?

Comment (pedrobaeza) [reply product_secondary_unit/migrations/11.0.1.0.2/post-migration.py]: For writing the group also on eventually deactivated users (that can be activated later, and being without the group).

Comment (sergio-teruel) [reply product_secondary_unit/migrations/11.0.1.0.2/post-migration.py]: Ok. Changes done

**Review by chienandalu — APPROVED** [review]
Looks good. Also tested on runbot :+1:

**Review by rafaelbn — APPROVED** [review]
Functionally tested in runbot :+1: 

Thanks @sergio-teruel

File: product_secondary_unit/views/product_views.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <field name="groups_id" eval="[(4, ref('product.group_uom'))]"/>
         <field name="arch" type="xml">
             <xpath expr="//page[@name='general_information']" position="inside">
-                <group string="secondary unit of measure">
+                <group string="secondary unit of measure" groups="product_secondary_unit.group_secondary_unit">
```
Comment (pedrobaeza) [inline product_secondary_unit/views/product_views.xml]: This shouldn't be done here, but on `groups_id` in the view (where you have right now the `product.group_uom` one).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Oks. changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**